AirCasting/SessionViews/SessionCartViewModel.swift

```diff
@@ -0,0 +1,47 @@
+// Created by Lunar on 29/07/2021.
+//
+
+import Foundation
+
+class SessionCartViewModel: ObservableObject {
+    private let sessionCartFollowing: SessionCartFollowing
+    @Published var isFollowing: Bool = false
+    
+    init(sessionCartFollowing: SessionCartFollowing) {
+        self.sessionCartFollowing = sessionCartFollowing
```

Comment:
here - add this code instead of defaulting to `false`:

```Swift
self.isFollowing = sessionCartFollowing == .following
```
(or whatever the enum value is for followed state)

Reply:
we don't have any enum here :/. I tried with this:   _self.session.followedAt != nil ? (isFollowing = true) : (isFollowing = false)_  but I cannot get session from anywhere. Could You please help me with this?

Reply:
@jankrzempek this will require some more structural changes to the solution (which would also making it cleaner). Take a look at my proposition : #126

Reply:
I went through it, awesome code. I imagine it would be hard for me to write it in that way at leat for now, but Thank You for showing me 😇